pkg/ovsdb/handler.go

```diff
@@ -409,12 +418,20 @@ func (ch *Handler) addMonitor(params []interface{}, notificationType ovsjson.Upd
 	if _, ok := ch.handlerMonitorData[jsonValueString]; ok {
 		return nil, fmt.Errorf("duplicate monitor ID")
 	}
+	schema, ok := ch.db.GetSchemas()[cmpr.DatabaseName]
```

Comment:
please rename `schema` to `databaseSchema`

Reply:
You may consider adding to libovsdb a method `LookupDatabase()` which already returns a correctly formatted error in case of failure

Reply:
in general, I think we need to change the GetSchemas() method. In transactions, you don't need all schemas too, transactions always work with a specific database.